src/pkg/packaging/windows/package.targets

```diff
@@ -22,7 +22,7 @@
             Condition="'$(OSGroup)' == 'Windows_NT' and '$(GenerateMSI)' == 'true'">
       
       <PropertyGroup>
-        <WixVersion Condition="'$(WixVersion)' == ''">3.10.2</WixVersion>
+        <WixVersion Condition="'$(WixVersion)' == ''">3.10.4</WixVersion>
```

Comment:
How can this ever happen when `$(WixVersion)` is unconditionally set in the `package.props` file that's imported above?

Can we just remove this line?